### PR TITLE
Layout & Responsiveness for Homepage

### PR DESCRIPTION
**Pull request by hemenge133**
Get this merged so theres no more conflicts. I'll do some finishing touches later.

### CODE REVIEW

**Review by scgaya — APPROVED** [review]
looks good!

File: client/src/components/SignUp/SignUp.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ class FormPage extends React.Component {
               </MDBBtn>
             </div>
           </form>
-          {/* <MDBRow>
```
Comment (xarielx) [inline client/src/components/SignUp/SignUp.js:89]: Everything looks good besides this, can you either comment it back out or delete this section, if not there will be two sign up forms

**Review by xarielx — CHANGES_REQUESTED** [review]
Just delete the second form on SignUp.js and we can merge this

File: client/src/components/SignUp/SignUp.js (hunk 1 above)
Comment (hemenge133) [reply client/src/components/SignUp/SignUp.js:89]: Removed it

**Review by xarielx — APPROVED** [review]
Looks good, let us hope it runs

**Pull request merged**